onadata/apps/api/management/commands/delete_users.py

```diff
@@ -0,0 +1,102 @@
+"""
+Delete users management command.
+"""
+import sys
+from django.contrib.auth.models import User
+from onadata.apps.logger.models import XForm
+from onadata.apps.logger.models import Instance
+from onadata.apps.logger.models import Project
+from django.core.management.base import BaseCommand, CommandError
+from django.utils import timezone
+
+
+def get_user_object_stats(username):  # pylint: disable=R0201
+    """
+    Get User information.
+    """
+    user = User.objects.get(username=username)
+    # Get the number of projects for this user
+    user_projects = Project.objects.filter(created_by=user).count()
+    # Get the number of forms
+    user_forms = XForm.objects.filter(user=user).count()
+    # Get the number of submissions
+    user_sumbissions = Instance.objects.filter(user=user).count()
+
+    result = {
+        'projects': user_projects,
+        'forms': user_forms,
+        'submissions': user_sumbissions
+    }
+
+    return result
+
+
+def inactivate_user(username):
+    """
+    Soft deletes the user termporarily.
+    """
+    try:
+        user = User.objects.get(username=username)
```

Comment:
This isn't really necessary.... but we could switch this get to `get(username=username, email=email)` so there is less chance of retrieving the wrong user.... Also kind of enforces the need for the `user_input` in the command being correct...

As this command should only delete the user when the user + email successfully retrieve a user... right ?

Reply:
yeah, this makes sense...caught that change. Thanks